newdle/client/src/styles/palette.scss

```diff
@@ -1,2 +1,4 @@
 $coral: #ff8383;
 $pink: #ff92c3;
+$purple: #6101b3;
+$grey: grey;
```

Comment:
I wonder if we need this. IMO it's better to have all the colors in the same place. If you we go for this, I think we're missing `white` that's being used in other files.

Reply:
Yes, let's have them all in one place

Reply:
Do you think I should declare all defaults in that file in the end?

Reply:
I'd declare in the palette the ones that we use. If later on we decide to change e.g the grey, we just have to modify this file.